src/views/sources/addSourceModal.tsx

```diff
@@ -121,7 +122,7 @@ const useSourceForm = ({
         name: name,
         credentials: credentials?.map(c => Number(c)),
         hosts: hosts?.split(','),
-        port: port || (isNetwork ? '22' : '443'),
+        port: port || (isOpenshift ? '6443' : isNetwork ? '22' : '443'),
```

Comment:
required: nested ternaries can create long term readability issues lets flatten it
```
port: port || (isOpenshift && 6443) || (isNetwork && '22') || '443',
```